filebeat/module/mongodb/log/ingest/pipeline.json

```diff
@@ -4,7 +4,9 @@
     "grok": {
       "field": "message",
       "patterns":[
-        "%{TIMESTAMP_ISO8601:mongodb.log.timestamp} %{WORD:mongodb.log.severity} %{WORD:mongodb.log.component} \\s*\\[%{WORD:mongodb.log.context}\\] %{GREEDYDATA:mongodb.log.message}"
+
```

Comment:
Can you remove this empty line?

Reply:
Sure I will give all this  build stuff a go if I can get a chance today.

Reply:
What is the error you are getting?
You need to put you example logs in `filebeat/module/mongodb/log/test/mongodb-debian-3.2.11.log`. Before running the generator command, you need to build filebeat using `make filebeat.test` and then run `GENERATE=1 INTEGRATION_TESTS=1 TESTING_FILEBEAT_MODULES=mongodb nosetests tests/system/test_modules.py`.